### PR TITLE
Add support for multiple projects in CSV file

### LINKED ISSUE #466: Allow multiple projects within a input single csv file
### Definition of Done / Acceptance Criteria

As `dri-to-pixl` using pixl, I want to be able to have multiple projects defined within a single input csv file

- [ ] more than one `extract_name` can be defined in the input csv and the populate code iterates through each of these, publishing as needed
- [ ] for omop input, throw an error if there are multiple projects 

### Testing

 - [ ] unit tests for multiple projects as input for csv (don't need for omop es

### Documentation

- update end user documentation to make it clear that multiple projects can be defined in a single csv file

### Dependencies

_No response_

### Details and Comments

may be worth renaming `load_from_parquet` to be `load_from_omop_parquet`?

### PR DESCRIPTION
**Pull request by p-j-smith**
## Description
Fixes #466 

- update `filter_exported_or_add_to_db` to handle filtering and adding images for multiple projects
- add function `filter_exported_or_add_to_db_for_project` to filter messages and add images for a single project
- add test case with CSV file that contains multiple projects
- update CLI docs to mention multiple projects can be defined in a CSV file

#466 also mentions that an error should be raised if omop input contains multiple projects, but as far as I can tell this is not possible as the project name [comes from the `extract_summary.json`](https://github.com/UCLH-Foundry/PIXL/blob/76b6a5f9142d894343fa22a8dd29ecca34ea952d/cli/src/pixl_cli/_io.py#L43). Is it possible that this field contains multiple project names, or is there another way omop input could contain multiple projects?

## Type of change
Please delete options accordingly to the description.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

### Suggested Checklist

- [X] I have performed a self-review of my own code.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [X] I have passed on my local host device. (see further details at the [CONTRIBUTING](https://github.com/UCLH-Foundry/PIXL/blob/main/CONTRIBUTING.md#local-testing) document)
- [X] Make sure your branch is up-to-date with main branch. See [CONTRIBUTING](https://github.com/UCLH-Foundry/PIXL/blob/main/CONTRIBUTING.md) for a general example to syncronise your branch with the `main` branch.
- [ ] I have requested PR review to `UCLH-Foundtry/arc-dev`
- [ ] I have adddressed and marked as resolved all the review comments in my PR.
- [ ] Finally, I have selected `squash and merge`

### CODE REVIEW

File: cli/README.md [hunk 1]
```diff
@@ -81,6 +81,16 @@ parquet_dir
...
+Alternatively, the queue can be populated based on records in CSV files:
+
+```bash
+pixl populate <path/to/file.csv>
+```
+
+One advantage of using a CSV file is that multiple projects can be listed
+for export in the file. Using the parquet format, in contrast, only supports
+exporting a single project per call to `pixl populate`.
+
```
Comment (p-j-smith) [inline cli/README.md:93]: I wasn't sure if this is the right place to add these docs, happy to move it

File: cli/tests/test_messages_from_files.py [hunk 2]
```diff
@@ -55,6 +55,26 @@ def test_messages_from_csv(omop_resources: Path) -> None:
     assert messages == expected_messages
 
 
+def test_messages_from_csv_multiple_projects(
+    omop_resources: Path, rows_in_session, mock_publisher
+) -> None:
+    """
+    GIVEN the database has a single Export entity, with one exported Image, one un-exported Image,
+    WHEN we parse a file with two Export entities, one new and one with two existing images and
+    one new image,
```
Comment (stefpiatek) [inline cli/tests/test_messages_from_files.py:64]: Tried to make it a bit more clear, but not sure if it actually is

```suggestion
    WHEN we parse a file with two projects, each with the same 3 images
      where one project has already exported one of the images 
      and the other project has not exported any images
```

**Review by stefpiatek — APPROVED** [review]
Lovely to see this in

File: cli/tests/test_messages_from_files.py (hunk 2 above)
Comment (p-j-smith) [reply cli/tests/test_messages_from_files.py:64]: nice, it's clearer to me (although I already know what the test does...)

**Pull request merged**